airbyte-integrations/connectors/source-opsgenie/source_opsgenie/schemas/teams.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": ["null", "string"]
```

Comment:
Use `number` as type instead of `string`

Reply:
Similar to above, the ID returned is a UUID:

{
    "data": [
        {
            "id": "02d0e242-d101-4a90-b9e2-c7da87fce0b1",
            "name": "Oncall - Template",
            "description": "",
            "links": {
                "web": "https://app.opsgenie.com/teams/dashboard/02d0e242-d101-4a90-b9e2-c7da87fce0b1/main",
                "api": "https://api.opsgenie.com/v2/teams/02d0e242-d101-4a90-b9e2-c7da87fce0b1"
            }
        },
        ...
    ],
    "took": 0.003,
    "requestId": "d6bdf2a0-2902-42f8-8ae4-bb064c9ce8e6"
}